README.md

```diff
@@ -172,3 +172,7 @@ It is acceptable to use the following section divider when it aids clarity. For
 ```python
 # -----------------------------------------------------------------------------
 ```
+
+## Variable names
+
+Do not use one letter variable names, except for `i` or `j` in loops. Using `k` or `v` instead of `key` or `value`, for example in a `dict` comprehension is never appropriate.
```

Comment:
quoting from the go doc:

> the further from its declaration that a name is used, the more descriptive the name must be

this makes total sense to me. so if you have

```py
f'{k}--{v}' for k, v in foo.items()
```

this is OK, but if you hava

```py
for k, v in foo.items():
   ... 20 lines block
```

then you probably want a more descriptive name